yw_website/yw_website/apps/website/views.py

```diff
@@ -80,11 +80,28 @@ def my_workflows(request):
     page = request.GET.get("page")
     workflows = paginator.get_page(page)
     host = request.get_host()
-
+    if request.method == "POST":
+        render(
+        request, "pages/edit_page.html", {"workflow_list": workflows, "host": host}
+    )
     return render(
-        request, "pages/home_page.html", {"workflow_list": workflows, "host": host}
+        request, "pages/my_workflows.html", {"workflow_list": workflows, "host": host}
     )
 
+def edit_workflow(request, workflow_id, version_id):
+    workflow = Workflow.objects.get(pk=workflow_id)
+    if request.method == "POST":
```

Comment:
Needs to save tags associated with Workflow using TagWorkflow

Reply:
Now that the workflow tags are acquired, you will have to use that relationship to get the Tag objects that are associated with the workflowTags, and then pass that to the html.